src/cgame/cg_ents.cpp

```diff
@@ -1373,6 +1373,18 @@ void CG_AddPacketEntities()
 			}
 		}
 	}
+
+	static Cvar::Range<Cvar::Cvar<float>>
+		cg_drawDebugDistance("cg_drawDebugDistance", "draw a circle around "
+				"yourself to give an idea of distances. Set this "
+				"to 0 to disable", Cvar::CHEAT, 0.0f, 0.0f,
+				5000.0f);
+	//make an attempt at drawing bounding boxes of selected entity types
```

Comment:
I don't understand. Is this about drawing a circle to estimate distance, or is it about BBoxes?

Reply:
Whoops, comment sneaked in. Removing